.gitmodules

```diff
@@ -16,6 +16,9 @@
 [submodule "layers/meta-intel"]
 	path = layers/meta-intel
 	url = https://git.yoctoproject.org/git/meta-intel
+[submodule "layers/meta-iot2000"]
+	path = layers/meta-iot2000
+	url = https://github.com/siemens/meta-iot2000.git
```

Comment:
This commit should be part of the other commit which adds the gitmodules: https://github.com/resin-os/resin-iot2000/pull/1/commits/0be5e793ceda3f5f24353bcca53cde167fd7fa8f

Please merge the 2